src/helpers.js

```diff
@@ -154,9 +153,25 @@ export function normalizeSwagger(parsedSpec) {
               if (!operation[inhName]) {
                 operation[inhName] = inherits[inhName]
               }
+              else if (inhName === "parameters") {
+                var exists = false
```

Comment:
@ponelat it's already addressed here https://github.com/swagger-api/swagger-js/pull/990/files#diff-42a353e98c906af524c7d3d48feaf72aR359